packages/ffe-icons-react/src/build-jsx-components.js

```diff
@@ -1,31 +1,57 @@
 import caseUtil from 'case';
+import cheerio from 'cheerio';
 import fs from 'fs';
+import path from 'path';
 import mkdirp from 'mkdirp';
-import icons from '../tmp/icons';
 
 mkdirp.sync('./jsx');
 
+const createSvgMap = () => {
+    const map = {};
+    const iconsPath = path.join(__dirname, '..', 'node_modules', '@sb1', 'ffe-icons', 'icons');
+    fs.readdirSync(iconsPath)
+        .filter(fileName => fileName.match(/\.svg$/))
+        .forEach((fileName) => {
+            const iconPath = path.join(iconsPath, fileName);
+            const iconName = fileName.split('.svg')[0];
```

Comment:
Or use [path.basename](https://nodejs.org/api/path.html#path_path_basename_path_ext):
```js
const iconName = path.basename(fileName, '.svg');
```

Reply:
Didn't know about that one!